types/src/errors/internal.rs

```diff
@@ -9,4 +9,12 @@ pub enum Error {
     Mailer(String),
     #[error("other: {0}")]
     Other(String),
+    #[error("rendering template: {0}")]
+    Template(String),
+}
+
+impl From<askama::Error> for Error {
```

Comment:
this should have `#[cfg(feature = "askama")]` above, and should be placed next to `From<validator::ValidationError>` impl.

Reply:
Done.